client/lib/domains/email-forwarding/index.js

```diff
@@ -1,14 +1,23 @@
 import emailValidator from 'email-validator';
 import { mapValues } from 'lodash';
+import { hasDuplicatedEmailForwards } from 'calypso/lib/domains/email-forwarding/has-duplicated-email-forwards';
 
-function validateAllFields( fieldValues ) {
+function validateAllFields( fieldValues, existingForwards = [] ) {
```

Comment:
Minor but it would be great to be consistent regarding naming (in this particular case, use the same term than in the `hasDuplicatedEmailForwards()` function:

```suggestion
function validateAllFields( fieldValues, existingEmailForwards = [] ) {
```